app/forms/add_to_order_form.rb

```diff
@@ -31,6 +31,7 @@ def save
 
     if @original_order.facility.id == @facility_id
       add_to_order!
+      return true
     end
 
     return true unless SettingsHelper.feature_on?(:cross_core_projects)
```

Comment:
I think I'd prefer to wrap the feature flagged changes in a conditional, instead of using a guard clause:
```
  if SettingsHelper.feature_on?(:cross_core_projects)
    if order_for_selected_facility.nil?
      create_cross_core_project_and_add_order!
    else
      add_to_order!
    end
  end
```
... then we can still git `true` on line 45.
This would be easier to read and clearer what needs to happen when we are ready to remove the flag.